dataactbroker/handlers/fileHandler.py

```diff
@@ -151,18 +151,13 @@ def getErrorReportURLsForSubmission(self, is_warning = False):
             # Unexpected exception, this is a 500 server error
             return JsonResponse.error(e, StatusCode.INTERNAL_ERROR)
 
-    def get_signed_url_for_submission_file(self):
+    def get_signed_url_for_submission_file(self, submission):
```

Comment:
Do we use this anywhere in this function? You took out the `user_agency_must_match` which I think was where the submission was needed.